shared/us-east-1/tools-costs-notifications/README.md

```diff
@@ -111,6 +111,10 @@ Before using this script, you need to set up the following:
 
 3. **Amazon SES Configuration**: Ensure that your AWS environment has Amazon SES configured with the appropriate permissions to send emails.
 
+| :point_up: Note   |
+|:---------------|
+| If you have SES in place but you've created a new identity (email remitent) VERIFY it in the [AWS SES Console](https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/identities)! |
```

Comment:
```suggestion
| If you have SES in place but you've created a new identity (email recipient) VERIFY it in the [AWS SES Console](https://us-east-1.console.aws.amazon.com/ses/home?region=us-east-1#/identities)! |
```

Reply:
And I would also suggest using a region-less URL or at least a note indicating that the URL needs to be adjusted to the region in-use.

Reply:
agreed